ondemandnavi: Switch to ALSA output

Update the talk scripts to use ALSA output via gst-launch-1.0 instead
of 4A's 4a-play.  gstreamer is used since it is likely that a further
revision will change to a pipewire output sink and add back setting a
role property.

Upstream-Status: pending

Signed-off-by: Scott Murray <scott.murray@konsulko.com>

diff --git a/package/flite b/package/flite
index 4c17b80..c837149 100755
--- a/package/flite
+++ b/package/flite
@@ -1,5 +1,4 @@
 #!/bin/sh
 TMP=/tmp/navi.wav
 echo "$1" | flite_hts_engine -m  /usr/share/Voice/us/cmu_us_arctic_slt.htsvoice -o $TMP
-4a-play $TMP hw:0,0,3 navigation
-
+gst-launch-1.0 filesrc location=$TMP ! decodebin ! audioconvert ! audioresample ! alsasink
diff --git a/package/jtalk b/package/jtalk
index ed82281..c075317 100755
--- a/package/jtalk
+++ b/package/jtalk
@@ -1,6 +1,4 @@
 #!/bin/sh
 TMP=/tmp/navi.wav
 echo "$1" | open_jtalk -ow $TMP -m /usr/share/Voice/mei/mei_normal.htsvoice -x /usr/share/dic/
-4a-play $TMP hw:0,0,3 navigation
-
-
+gst-launch-1.0 filesrc location=$TMP ! decodebin ! audioconvert ! audioresample ! alsasink
